preprocessing/nextclade/src/loculus_preprocessing/backend.py

```diff
@@ -221,17 +222,32 @@ def upload_embl_file_to_presigned_url(content: str, url: str) -> None:
         raise RuntimeError(msg)
 
 
-def download_minimizer(url, save_path):
-    try:
-        response = requests.get(url, timeout=10)
-        response.raise_for_status()
+def download_minimizer(config: Config, results_dir: str) -> None:
+    for segment in config.nucleotideSequences:
+        if config.minimizer_url:
+            url = config.minimizer_url
+        elif get_nextclade_dataset_server(config, segment) is not None:
+            url = (
+                get_nextclade_dataset_server(config, segment).rstrip("/") + "/minimizer_index.json"
+            )
```

Comment:
Hmm, I'm not sure it's worth doing this now on main as opposed to just on the tip of multipath. Why are we downloading minimizers for each segment? I thought we always use a single minimizer per organism?

Reply:
I realized its then ambiguous which server is used as a fallback option